src/plugins/peer-view/public/js/peerview.js

```diff
@@ -120,9 +128,13 @@
       self.connecting = false;
     });
     var buffer = [];
+    var rovBlacklistMessages = [
+      'newListener',
+      'sys.pong'
+    ]
```

Comment:
There are some messages coming from the rov socket.io connection that we don't want to forward to the webrtc peers.  We are blocking the pong response from the ROV to the host from going forward because the peer code does not have the patching ping record and would not calculate the latency correctly.